### ISSUE TITLE
Demand forecasting is not good

### ISSUE THREAD

**Issue by dara1400** [moment-timeseries-foundation-model/moment#21]
I used a dataset of a demand of electric load for one month. The step is one hour. The values are normalized.
I feed 512 point to the model and forecasting horizon is 24.
The output is not good.
![image](https://github.com/moment-timeseries-foundation-model/moment/assets/31727520/c3cb1bff-9980-4af1-97b4-2d8f52aaa918)

**Comment by dara1400** [comment]
This another simple test:

![image](https://github.com/moment-timeseries-foundation-model/moment/assets/31727520/9515aa65-c379-4872-bfe5-75efa466acc6)

**Comment by mononitogoswami** [comment]
Hi Hamzeh, 
Thanks for your interest in MOMENT! As we have mentioned in the [forecasting tutorial](https://github.com/moment-timeseries-foundation-model/moment/blob/main/tutorials/forecasting.ipynb), MOMENT is pre-trained to reconstruct input time series. To use MOMENT for forecasting, we replace the reconstruction head with a forecasting head-- a linear layer which maps MOMENT embeddings to the forecasting horizon. This forecasting head is **randomly initialized and must be fine-tuned before use**. We have made this clearer in the tutorial now 🤗

While MOMENT can do zero-shot reconstruction, imputation, anomaly detection, and unsupervised representation learning for classification, it does not do zero-shot forecasting. 

I would recommend fine-tuning MOMENT on your data for a few epochs. You can find example fine-tuning code in the [forecasting tutorial](https://github.com/moment-timeseries-foundation-model/moment/blob/main/tutorials/forecasting.ipynb). As you can see from the tutorial, with only 1 epoch and no hyperparameter tuning, MOMENT can get competitive forecasting results!

I'll close this issue for now, but please feel free to re-open this issue, or create another one if you face more problems! Again, thank you so much for your interest in MOMENT!
